app/views/decidim/devise/sessions/new.html.erb

```diff
@@ -1,73 +0,0 @@
-<% add_decidim_page_title(t("devise.sessions.new.sign_in")) %>
```

Comment:
The login page was a bit of an eyesore. This looks good!

<img width="1680" alt="Screenshot 2020-07-29 14 22 17" src="https://user-images.githubusercontent.com/9510/88854992-0dd49280-d1a7-11ea-923b-60a5ab321022.png">